electrum/gui/kivy/uix/dialogs/__init__.py

```diff
@@ -213,13 +213,18 @@ def update(self, outputs: Sequence['TxOutput']):
         res = []
         for o in outputs:
             value = self.app.format_amount_and_units(o.value)
-            res.append({'address': o.get_ui_address_str(), 'value': value})
+            res.append({
+                    'address': o.get_ui_address_str(),
+                    'value': value,
+                    'background_color': (0.3, 0.3, 0.3, 1),
+                    'color': (1, 1, 1, 1)})
```

Comment:
why are these colors needed to be specified here?
when are they used?

Reply:
Thanks for pointing this out. I couldn't find the definition of the kivy default colors, and therefore had to state them here again. I would like to improve this if I can. Do you know where the kivy default colors are defined?

Reply:
I removed these 2 lines and it still works.  That seems cleaner.

Reply:
@SomberNight : done.